decidim-core/vendor/assets/javascripts/datepicker-locales/foundation-datepicker.lv.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Latvian translation for foundation-datepicker
+ */
+;(function($){
+	$.fn.fdatepicker.dates['lv'] = {
+		days: ["Svētdiena", "Pirmdiena", "Otrdiena", "Trešdiena", "Ceturtdiena", "Piektdiena", "Sestdiena", "Svētdiena"],
```

Comment:
Here there is the word "Sunday" written twice, we took inspiration on the french day [foundation-datepicker](https://github.com/decidim/decidim/blob/develop/decidim-core/vendor/assets/javascripts/datepicker-locales/foundation-datepicker.fr.js)
Is it necessary to define "sunday" multiple times ?

Reply:
I see that many languages have "Sunday" written twice, but some languages like Catalan or Estonian or Basque not:
- https://github.com/decidim/decidim/blob/develop/decidim-core/vendor/assets/javascripts/datepicker-locales/foundation-datepicker.ca.js
- https://github.com/decidim/decidim/blob/develop/decidim-core/vendor/assets/javascripts/datepicker-locales/foundation-datepicker.et.js
- https://github.com/decidim/decidim/blob/develop/decidim-core/vendor/assets/javascripts/datepicker-locales/foundation-datepicker.eu.js

Reply:
Thank you for this answer, I removed the duplicated words in the latvian datepicker file